html/admin/api/notifications/slack/slack.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require_once __DIR__ . '/../../apiHead.php';
+function sendSlackNotification($user, $subject)
+{
+    global $DBLIB,$CONFIG;
+
+    $DBLIB->where("users_userid", $user["userData"]['users_userid']);
+    $userSlack = $DBLIB->getone("users",["users_oauth_slackid"]);
+    $channel_id = $userSlack['users_oauth_slackid'];
+
+    if (!$user or $channel_id == '') return false; //If the user hasn't linked slack yet
+
+    $token = $CONFIG['SLACK_BOT_OAUTH'];
+
+    // TODO add more detailed templates for slack notifications, will involve looking at the notify function because atm it only deals with email templates
+    if ($token != null) {
+        $data = array(
+            "channel" => $channel_id,
+            'type' => 'plain_text',
+            'text' => $subject
+        );
```

Comment:
There's a commit showing how to make more complicated messages, it works but the preview breaks, so until we add templates this is the most elegant option